server/src/main/scala/sbt/server/ServerLogging.scala

```diff
@@ -26,7 +26,12 @@ private[sbt] abstract class EventLogger[E <: protocol.LogEvent](protected val lo
   }
 
   def log(level: Level.Value, message: => String): Unit = {
-    send(protocol.LogMessage(level.toString, message))
+    // Here we're trying to catch when the message already has the log level in it, for some reason.
+    val m = message
+    val msg = logLevelRegex.findFirstMatchIn(m).flatMap { m =>
+      Some(m.group("message"))
+    } getOrElse (m)
```

Comment:
HA! A few too many references to an identifier `m`.  It's all good, but perhaps a little more creativity on the identifier selection. :-)